woocommerce/woocommerce-free-shipping-progress-bar-block
# Make free shipping from value editable

Fixes #9 

## Note

In i1 of this block, the merchant must be able to provide the free shipping value in the editor via the Inspector Controls. This PR adds this functionality.

<table>
<tr>
<td>Before:
<br><br>

![#9-before](https://user-images.githubusercontent.com/3323310/134641775-70a39e0d-f838-44b1-98e6-d68f520a5d60.png)
</td>
<td>After:
<br><br>

![#9-after](https://user-images.githubusercontent.com/3323310/134641752-5eaccd08-1ceb-4e9c-bf75-5854764b08a9.png)
</td>
</tr>
</table>

## Test instructions

- Check out this PR.
- Create a test page.
- Add the block `Free Shipping Progress Bar` to the test page.
- Click on the block to open the Inspector Controls.
- Adjust the free shipping value to ensure that the progress bar changes.
